lib/python/asterisk/realtime_converter.py

```diff
@@ -110,7 +113,7 @@ def write_sorcery_conf(self):
         Keyword Arguments:
         config_dir: The directory where Asterisk configuration can be found
         """
-        with open(self.sorcery.file, 'a') as sorcery:
+        with open(self.sorcery.file, 'a+') as sorcery:
```

Comment:
What was the reason for adding the "+" to the "w" and "a" opens?  The files aren't read from in the scope of the opens.